components/dropdown/src/auro-dropdown.js

```diff
@@ -236,6 +277,12 @@ export class AuroDropdown extends LitElement {
     if (changedProperties.has('mobileFullscreenBreakpoint')) {
       this.bibContent.mobileFullscreenBreakpoint = this.mobileFullscreenBreakpoint;
     }
+
+    if (this.triggerContentSlot) {
+      this.hasTriggerContent = this.triggerContentSlot.reduce((old, sl) => old || Boolean(sl.textContent.trim()), false);
```

Comment:
Can you add some context around this logic?

Reply:
replaced to call `handleTriggerContentSlotChange`